### PR TITLE
Add check/install for prerequisite

### PR DESCRIPTION
**Pull request by jimmyliao**
Signed-off-by: Jimmy <jimmyliao@jimmyliao.net>

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**

```
# check all prerequisite
./install/primehub-prerequisite check

# check and install all prerequisite libraries if not installed
./install/primehub-prerequisite install
```

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by LiuYuWei** [comment]
@jimmyliao Finish reviewing the code. Please help to check the comments.

**Review by LiuYuWei — COMMENTED** [review]
@jimmyliao Please check.

File: install/primehub-prerequisite [hunk 1]
```diff
@@ -128,18 +151,197 @@ check::storageclass() {
...
+
+}
+
+install::kubectl() {
+  info "[Install] check kubectl..."
+
+  if [[ "$(command -v kubectl)" == "" ]]; then
+    local platform=$(uname | tr '[:upper:]' '[:lower:]')
+    info "[Download] kubectl ${KUBECTL_VERSION}"
+    curl -O -L "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/${platform}/amd64/kubectl" && chmod +x ~/bin/kubectl && ln -nfs ~/bin/kubectl ~/bin/k
```
[the remark this answers is not in the document]
Comment (jimmyliao) [reply install/primehub-prerequisite:172]: already achieve line 307

**Comment by jimmyliao** [comment]
> @jimmyliao Please check.

@LiuYuWei , revised per commit #ce38ef8. Please check

**Pull request merged**